distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Dialect_Menu.enso

```diff
@@ -0,0 +1,14 @@
+from Standard.Base import all
```

Comment:
I think if this file has nothing to do with GUI/frontend, then perhaps call it "Dialect_Variants" or "SQL_Variants". Or even just move it into Base_Generator.

Reply:
Yeah I wonder if just in Base_Generator makes sense for now. Just creates a weird dependency inversion that I was trying to avoid where we bounce back and forth between the Base_Generator and the Dialects

Reply:
Changed it. Think that is cleaner